core/src/main/java/nu/yona/server/subscriptions/entities/UserPrivate.java

```diff
@@ -145,9 +145,9 @@ public void removeBuddy(Buddy buddy)
 		UserAnonymized.getRepository().save(userAnonymized);
 	}
 
-	public void removeBuddiesFromUser(UUID fromUserLoginID)
+	public void removeBuddiesFromUser(UUID fromUserID)
```

Comment:
There will be only one buddy object for a given user ID (these days). I'm renaming this into ``removeBuddyForUserID(UUID userID)``.